microcosm_flask/conventions/build_info.py

```diff
@@ -42,7 +42,6 @@ def build_info():
 
 
 @defaults(
-    path_prefix="",
```

Comment:
In practice, we don't use the ability to re-prefix the convention-driven endpoints; instead, we use a different `Namespace`.

Removing this "feature" helps clarify the `Namespace` API, which is currently a little confusing.